tests/test_sample.py

```diff
@@ -85,14 +85,10 @@ def test_sample_values_hermitian(self, device, shots, tol):
         assert np.allclose(sorted(list(set(s1))), sorted(eigvals), **tol)
 
         # the analytic mean is 2*sin(theta)+0.5*cos(theta)+0.5
-        assert np.allclose(
-            np.mean(s1), 2 * np.sin(theta) + 0.5 * np.cos(theta) + 0.5, **tol
-        )
+        assert np.allclose(np.mean(s1), 2 * np.sin(theta) + 0.5 * np.cos(theta) + 0.5, **tol)
```

Comment:
I'm assuming this is just because they were previously blacked without `-l 100`.

Reply:
Yes, it appears so